jetstream/cli.py

```diff
@@ -746,8 +746,11 @@ def rerun(
         )
 
     # update table timestamps which indicate whether an experiment needs to be rerun
+    client = BigQueryClient(project_id, dataset_id)
     for slug in experiment_slug:
-        BigQueryClient(project_id, dataset_id).touch_tables(slug)
+        client.touch_tables(slug)
+        # delete existing tables
+        client.delete_experiment_tables(slug, analysis_periods, not recreate_enrollments)
```

Comment:
Should this be `recreate_enrollments` instead of `not recreate_enrollments`?

Reply:
Yes, you are right. Good catch